server/serializers/subscriptions.go

```diff
@@ -14,6 +14,10 @@ type UserID struct {
 	UniqueName  string `json:"uniqueName"`
 }
 
+type PublisherInputsGeneric struct {
+	ProjectID string `json:"projectId"`
+}
+
 type PublisherInputsBoards struct {
 	ProjectID string `json:"projectId"`
```

Comment:
```suggestion
	PublisherInputsGeneric
```
See if this works and if it does, do this for repos too.

Reply:
It didn't work due to a mixture of struct and fields
error message: `mixture of field:value and value elements in struct literal`